### PR TITLE
Translated the new strings in Greek

### PR DESCRIPTION
**Pull request by Happy3450**

### CODE REVIEW

File: el-GR.json [hunk 1]
```diff
@@ -1008,20 +1008,20 @@
 	"STREMIO_TV_STREAMS_STILL_LOADING": "Το ${1} ακόμα φορτώνει ...",
 	"STREMIO_TV_PLAYER_SETTINGS": "Ρυθμίσεις προγράμματος αναπαραγωγής",
 	"STREMIO_TV_PLAYER_SUBTITLES_DELAY": "Καθυστέρηση υποτίτλων",
-	"STREMIO_TV_PLAYER_AUDIO_DELAY": "Audio delay",
-	"STREMIO_TV_PLAYER_VIDEO_SCALING": "Video scaling",
+	"STREMIO_TV_PLAYER_AUDIO_DELAY": "Καθυστέρηση ήχου,
```
Comment (jaruba) [inline el-GR.json:1011]: there is a missing double quote before the comma at the end.

Comment (Happy3450) [reply el-GR.json:1011]: Sorry, I'll fix that.

Comment (Happy3450) [reply el-GR.json]: I fixed it but it isn't shown.

**Review by jaruba — APPROVED** [review]

**Comment by jaruba** [comment]
@Happy3450 thanks!

**Pull request merged**